src/lib/runtime/overlayfs/overlayfs.c

```diff
@@ -63,7 +63,7 @@ int _singularity_runtime_overlayfs(void) {
     } else if ( singularity_registry_get("WRITABLE") != NULL ) {
         singularity_message(VERBOSE3, "Not enabling overlayFS, image mounted writablable\n");
     } else {
-#ifdef SINGULARITY_OVERLAYFS
+#if (SINGULARITY_OVERLAYFS || SINGULARITY_TRY_OVERLAYFS)
```

Comment:
According to above, it seems that SINGULARITY_OVERLAYFS will never be defined. Is that accurate?

Reply:
No, why would you say that?  configure will define it if modules.dep exists and the overlay module is in it (as before), or it will define SINGULARITY_TRY_OVERLAYFS if modules.dep does not exist.  Undefined variables are treated as 0 in an #if statement, and defined variables as 1.